packages/sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft/teamsfx",
-  "version": "3.0.2",
+  "version": "4.0.0",
```

Comment:
I think this should be 4.0.0-alpha.0, the CD pipeline will bump it to 4.0.0 together with templates. Please double check with @wenytang-ms